### PR TITLE
Cleaning up the mess

### PR DESCRIPTION
**Pull request by KC2016**
Deleting folders and files from DS from the main branch.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+~$*.xls*
+
+# Excel Backup File
+*.xlk
+
+# PowerPoint temporary
+~$*.ppt*
+
+# temp requirements Python env
+requeriments.txt
```
Comment (larry-dalmeida) [inline .gitignore:30]:
```suggestion
requirements.txt
```

Typo :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# system files
+.DS_Store
+.ipynb_checkpoints
+venv
+.venv
+
+# folder
+temp/a
```
Comment (larry-dalmeida) [inline .gitignore:8]: Why not just `temp/`? do you only want to ignore the `temp/a` subdirectory?

**Review by larry-dalmeida — APPROVED** [review]
I think there is a `.DS_Store` file being checked in also which may not be your intention. Otherwise, the PR looks good to me :)

File: .gitignore (hunk 2 above)
Comment (KC2016) [reply .gitignore:8]: Another typo. How do I fix this?

File: .gitignore (hunk 1 above)
Comment (KC2016) [reply .gitignore:30]: Thanks, should I commit suggestion directly in GitHub?

File: .gitignore (hunk 2 above)
Comment (KC2016) [reply .gitignore:8]: `.DS_Store` is a file created by VS code. Joanna has the same. It should me hidden, as far as I know.

Comment (larry-dalmeida) [reply .gitignore:8]: Yes, I usually add it to my ignore file

**Pull request merged**